examples/test_util/data/EclIOdata.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+  Copyright 2019 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPM_ECLIO_DATA_HPP
+#define OPM_ECLIO_DATA_HPP
+
+
+namespace examples {
+
+ namespace test_util {
+
+  namespace data {
+
+    const int sizeOfInte =  4;    // number of bytes pr integer (inte) element
+    const int sizeOfReal =  4;    // number of bytes pr float (real) element
+    const int sizeOfDoub =  8;    // number of bytes pr double (doub) element
+    const int sizeOfLogi =  4;    // number of bytes pr bool (logi) element
+    const int sizeOfChar =  8;    // number of bytes pr string (char) element
+
+    const int MaxNumInte = 1000;    // maximum number of integer elements in binary data block, 4000 bytes, 4 bytes pr element  
```

Comment:
The name `Max` is misleading - these number refer to block/record sizes and not a maximum as such.

Reply:
It is actually a maximum, the last block of data will (in most cases) not be of this size. Example 

ARRAY1  2500 INTE 

will be split into 3 block 1000, 1000 and 500. Both 500 and 1000 is block sizes. 1000 is maximum block size